frontend/src/components/dashboard/create/create-modal/CreateModal.tsx

```diff
@@ -68,16 +78,42 @@ const CreateModal = () => {
           </PrimaryButton>
         </div>
 
-        <p className={styles.subtext}>
-          Get your credentials ready.
-          <OutboundLink
-            eventLabel={GUIDE_CREDENTIALS_URL}
-            to={GUIDE_CREDENTIALS_URL}
-            target="_blank"
-          >
-            What is this?
-          </OutboundLink>
-        </p>
+        {selectedChannel === ChannelType.Email && (
+          <div className={styles.protectedOption}>
+            <i
```

Comment:
seems hard to click on mobile view. perhaps wrap the the checkbox and the text in an element that is clickable